cmd/ingest/batch.go

```diff
@@ -0,0 +1,199 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"sync/atomic"
+
+	ctx509 "github.com/google/certificate-transparency-go/x509"
+	"github.com/netsec-ethz/fpki/pkg/common"
+	"github.com/netsec-ethz/fpki/pkg/db"
+	mcommon "github.com/netsec-ethz/fpki/pkg/mapserver/common"
+	"github.com/netsec-ethz/fpki/pkg/mapserver/updater"
+)
+
+const BatchSize = 10000
+
+type Batch struct {
+	Certs  []*ctx509.Certificate
+	Chains [][]*ctx509.Certificate
+	cns    []*string
```

Comment:
Should be `[][]*string` since each certificate can contain multiple domains (CN + SANs)